src/actions/fetch/copier.ts

```diff
@@ -107,12 +81,11 @@ export class CopierRunner {
       copierValues.push("--data")
       copierValues.push(key + "=" + value)
     }
-    const templateSource = templateDir+"/copier"
     const projectDestination = intermediateDir+"/copier"
     if (copierInstalled) {
       await executeShellCommand({
         command: 'copier',
-        args: ['copy', ...copierValues, templateSource, projectDestination, '--trust'],
+        args: ['copy', ...copierValues, templateContentsDir, projectDestination, '--trust'],
         logStream,
       });
     } else {
```

Comment:
I'm not convinced the "run in container" part will work either, as we don't intend to run that way, please can we delete the ability to do that?

Reply:
should we still leave a check that copier is installed and to throw an error if it isn't?
Or should we assume that whoever uses this plugin will have copier installed? - Actually I'll get rid of the check and modify the docs to remove the notes on having the defined container option and that copier is compulsory / words to that effect.